### DISCUSSION TITLE
通知一覧で「いいね」をいただいた方のプロフィールへのリンクが欲しい

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#531] [要望・アイデア・不具合]
いつもありがとうございます。

現在は、通知一覧での「いいね」のリンクは、自分自身の記事へのリンクになっています。
これに加えて、「いいね」をしていただいた方（アイコンが出ている方）のプロフィールへのリンクが欲しいです。
どんな方が「いいね」してくれたのかが知りたくて、いつも、アイコン左のユーザ名をコピーして、`qiita.com/ユーザ名`を手打ちしています。（数が少ないので）

![通知一覧](https://github.com/increments/qiita-discussions/assets/42625437/da960bda-3ad7-46b0-b67e-58d44d6d6f0a)

「コメント」や「メンション」も同じ仕様で構わないと思います。
ご検討よろしくお願いします。

**Comment by shiracamus** [comment]
通知ベルアイコンからプルダウンした状態でユーザアイコン部分をクリックするといいねしてくれた方のプロフィールが表示されるので、通知一覧でもそうなるといいですね。

**Reply by nak435** [reply]
↑そのインタフェース、知りませんでした。
コメントありがとうございました。

**Reply by tanaka-asami415** [reply]
@nak435 
こちらこそ、いつもありがとうございます！
アクションをしてくれたユーザーページへのリンクがある方がいいと思うので対応します！
issueも作成しました https://github.com/increments/qiita-discussions/issues/535

@shiracamus 
フォローありがとうございます！🙏

**Reply by tanaka-asami415** [reply]
@nak435 
お待たせいたしました！

通知一覧からユーザーページへのリンクを追加しました！
https://qiita.com/release-notes/2023-09-19

構造的にユーザー名部分をリンクにするのは難しかったため、ユーザーアイコンにのみリンクを追加しています🙇
いいね以外にも、ストックや編集リクエストなどの通知もリンクを反映しました！

**Reply by nak435** [reply]
確認しました。ありがとうございました。
